flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -475,6 +477,11 @@ public boolean isChainingEnabled() {
         return isChainingEnabled;
     }
 
+    @PublicEvolving
+    public boolean isChainingOfOperatorsWithDifferentMaxParallelismEnabledEnabled() {
```

Comment:
You will also have to extend the Scala API if you're adding a method here.

Reply:
Not sure if we need really this method though, but I can see it's value w.r.t. consistent configuration.

(although it would be great if we would consolidate all these setters into some pojo, instead of having to modify the StreamExEnv whenever we expose another thing).